frontend/src/login.html

```diff
@@ -7,9 +7,9 @@
     <meta name="apple-mobile-web-app-capable" content="yes" />
 
     <!-- Chrome, Opera, and Firefox OS -->
-    <meta name="theme-color" content="#3a3f51" />
+    <meta name="theme-color" content="#2193B5" />
```

Comment:
This was intentionally dark, instead of using the brighter colour, I don't recall the original discussion, but the darker colour is going to be less abrasive for most people. What's the reason to change it?